contracts/Collectable.sol

```diff
@@ -2,38 +2,86 @@ pragma solidity ^0.4.21;
 
 import "./Authorizable.sol";
 
+/** @title Gets details of the subscription and terminates it after detecting insufficient tokens */
+/** @author Kerman Kohli - <kerman@TBD.com> */
+
 contract Collectable is Authorizable {
 
+    /**
+      * Public view functions
+    */
+
+    /** @dev Checks if the subscription is valid.
+      * @param _subscription is the identifier of the customer's subscription with it's relevant details.
+      * @return  success is the result of whether the subscription is valid or not.
+    */
+
     function isValidSubscription(bytes32 _subscription)
         view
         public
         returns (bool success);
 
+    /** @dev Gets the token contract address within the subscription.
+      * @param _subscription is the identifier of the customer's subscription with it's relevant details.
+      * @return subscriptionTokenAddress is token contract address within the subscription.
+    */
+
     function getSubscriptionTokenAddress(bytes32 _subscription)
         view
         public
         returns (address subscriptionTokenAddress);
 
+    /** @dev Gets the address from address and to address within the subscription.
+      * @param _subscription is the identifier of the customer's subscription with it's relevant details.
+      * @return from is the address where the recurring payments will be deducted from (the customer).
```

Comment:
from is the address where recurring payments are deducted (customer)
to is the address where recurring payments are sent (business)